doc/en/api/1.0.0/gwcseed.yaml

```diff
@@ -0,0 +1,138 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Seeding and Truncating
+  description: A layer is a published resource (feature type or coverage).
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /seed.json:
+    get:
+      operationId: seedGet
+      summary: Query's currently running GWC task
+      description: Query's and returns a json array of the status for all currently running task. Requires .json extension in the request. The array contains a set of long in the following order:[tiles processed, total number of tiles to process, number of remaining tiles, Task ID, Task status].
+        The returned task status will be one of -1 = ABORTED, 0 = PENDING, 1 = RUNNING, 2 = DONE
+      produces:
+        - text/json
```

Comment:
You have `text/json` here, but `application/json` elsewhere. Can you confirm that GWC really uses different media types for JSON for different endpoints?